intro.md

```diff
@@ -859,7 +859,7 @@ The following qualifiers are also supported in the search term:<br />
 
 <ul>
 <li><code>ids:key_id,...</code> for queries on a comma-separated list of ids</li>
-<li><code>name:key_name</code> for text queries on exact key names - whitespaces need to be prefixed with a backspace ("\")</li>
+<li><code>name:key_name</code> for text queries on exact key names - whitespaces, commas, and colons  need to be escaped with a backslash</li>
```

Comment:
Looks like there's double spaces after "colons". Shouldn't it be "double backslashes"?

Reply:
Thanks. Both should be fixed